src/modules/chat/index.js

```diff
@@ -363,6 +363,16 @@ export default class Chat extends Module {
 			}
 		});
 
+		this.settings.add('chat.filtering.highlight-basic-users-temp', {
```

Comment:
There's a `settings.addUI(key, definition)` method that should be used when adding UI to the settings menu that doesn't need an underlying setting.